tests/integration/baseTestAPI.py

```diff
@@ -159,6 +162,21 @@ def setUpClass(cls):
 
             sess.commit()
 
+            # todo: refactor below when we figure out best way for enums in help tables
+            # Create lookup dictionaries
+            js = sess.query(JobStatus)
+            cls.jobStatusDict = {j.name: j.job_status_id for j in js.all()}
+            jt = sess.query(JobType)
+            cls.jobTypeDict = {j.name: j.job_type_id for j in jt.all()}
+            ft = sess.query(FileType)
+            cls.fileTypeDict = {f.name: f.file_type_id for f in ft.all()}
+            fs = sess.query(FileStatus)
+            cls.fileStatusDict = {f.name: f.file_status_id for f in fs.all()}
+            sev = sess.query(RuleSeverity)
+            cls.ruleSeverityDict = {s.name: s.rule_severity_id for s in sev.all()}
+            et = sess.query(ErrorType)
+            cls.errorTypeDict = {e.name: e.error_type_id for e in et.all()}
+
```

Comment:
Should this instead use the dicts attached to the model objects?  For example, JobStatus has JOB_STATUS_DICT.  These can also be used with the BaseInterface.getIdFromDict function, which will use the dictionary when present and populate it if it is empty (i.e. on the first call).  That function is also used by the various helper functions in the interface objects that provide these values, like JobTrackerInterface.getJobStatusId (also has getJobTypeId and getFileTypeId).  I understand that the goal here is to use a different way of getting the session object, but I'm wondering if our approach should be to have a way to provide that session to the interface objects rather than duplicating those functions whenever they're needed outside the app.  Not too big a deal on this one, but may be worth thinking about.